YDotNet/Native/Types/Events/EventChangeNative.cs

```diff
@@ -1,40 +1,28 @@
 using System.Runtime.InteropServices;
-using YDotNet.Document.Cells;
-using YDotNet.Document.Types.Events;
 using YDotNet.Infrastructure;
 using YDotNet.Native.Cells.Outputs;
 
 namespace YDotNet.Native.Types.Events;
 
 [StructLayout(LayoutKind.Sequential)]
-internal struct EventChangeNative
+internal readonly struct EventChangeNative
 {
     public EventChangeTagNative TagNative { get; }
 
     public uint Length { get; }
 
     public nint Values { get; }
 
-    public EventChange ToEventChange()
+    public nint[] ValuesHandles
```

Comment:
@SebastianStehle Out of curiosity: why did you decide to use a property here and a method for almost all other event data?

It's internal but I'm still asking this because it seems inconsistent. I suggest we always use only one or only another approach, this way we can write similar code from now on.

Examples of method usages are available at `StateVectorNative` and `DeleteSetNative`.

Reply:
I have described it in the decisions.md

Reply:
Alright, so the rule of thumb is:

- If the data is contained inside the `struct`: use properties (when there's a known number of properties, like `nint`, `byte`, and other primitive types)
- If the data is outside the `struct`: use methods (for example, to read the data that's accessible via an `nint Handle` property and has size `uint Length`)

Right? If so, I understand it here. I'll try to make this more explicit in the `decisions.md` file.